### DISCUSSION TITLE
Feature request: Add Org Wide Variables

### DISCUSSION THREAD

**Discussion by na-jakobs** [community/community#17274] [Actions]
Dear GitHub,

It would be awesome to have the ability to add Org Variables the same was as you add Org Secrets. The only difference is that they are not masked or hidden in any way. Also Repo Vars.

Please consider this feature as we used it alot in our old Git solution and need it badly!

@chrispat 🙏

**Comment by mikeschinkel** [comment]
Yes! Please!
